index.js

```diff
@@ -869,6 +869,19 @@ micromatch.compilers = compilers;
 micromatch.parsers = parsers;
 micromatch.caches = cache.caches;
 
+/**
+ * Replace backslashes used as path separator with forward slashes.
+ * This is useful when doing `path.resolve` etc. in Windows.
+ */
+
+micromatch.replaceSeparatorsForGlob = function(path) {
+  if (typeof path !== 'string') {
+    throw new TypeError('expected a string: "' + util.inspect(path) + '"');
+  }
+
+  return path.replace(/\\(?![{}()+?.^$])/g, '/');
```

Comment:
copied from the linked Jest PR, all credit goes to @aldarund

Reply:
We might want to use this regex in the [toPosixSlashes](https://github.com/micromatch/picomatch/blob/master/lib/utils.js#L11) utility method in picomatch. However, I think we'll need to remove the `.`, since that would cause dotfiles and directories like `foo\\.git` to incorrect. I think we also need `*` to be added to the regex.